features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Comments/CommentAdd.jsx

```diff
@@ -130,7 +130,7 @@ class CommentAdd extends React.Component {
                 })
                 .catch((error) => {
                     console.error(error);
-                    if (error.response) {
+                    if (error.response.body.message) {
```

Comment:
can't the `response` or `body` become null here? that can throw something similar to NPE?

Reply:
what if error.response and error.response.body.message are both checked?

if (error.response && error.response.body.message) {...

Reply:
and error.response.body too.